sktime/split/expandingcutoff.py

```diff
@@ -36,53 +36,85 @@ def _validate_cutoff(cutoff):
 
 
 class ExpandingCutoffSplitter(BaseSplitter):
-    """
+    r"""
     Expanding cutoff splitter for time series data.
 
-    This splitter combines elements of `ExpandingWindowSplitter` and `CutoffSplitter`
-    to create training and testing sets. Unlike `ExpandingWindowSplitter` which begins
-    with a fixed initial window, this splitter uses a specific cutoff point as the
-    starting window for the training set. The training set then expands incrementally
+    This splitter combines elements of ``ExpandingWindowSplitter`` and
+    ``CutoffSplitter`` to create training and testing sets. Unlike
+    ``ExpandingWindowSplitter`` which begins with a fixed initial window,
+    this splitter uses a specific cutoff point as the starting window
+    for the training set. The training set then expands incrementally
     in each split until it reaches the end of the series.
 
-    The test set is defined by a forecast horizon relative to the last point in the
-    training set, containing as many subsequent indices as specified by the `fh`
-    parameter.
+    The test set is defined by a forecast horizon (``fh``) relative
+    to the last point in the training set, containing the specified
+    number of subsequent indices.
 
     The valid types of y-index and cutoff pairings are datelike-datelike, datelike-int,
     and int-int. When a datelike index is combined with an int cutoff, the cutoff
     functions as an iloc indexer. When an int index is paired with a positive int
     cutoff, the cutoff serves as a loc indexer. If the int cutoff is negative, it
     functions as an iloc indexer.
 
+    The traning folds are defined as follows: the training set
+    starts from the beginning of the series and extends up to
+    the ``cutoff`` index for the first split. In each subsequent
+    split, the training set expands by ``step_length``,
+    including all prior data up to the new cutoff.
+
+    .. math::
+        \text{TrainingFolds} = \{ [j, \ldots, cutoff + j]\}_{j=0, \ldots, N + 1}
```

Comment:
this starts at j, is that correct? Should it not be the first time point?

Further, it might not be too clear that you are using indices. That may need some explanation, or, alternatively, introduction of a time series such as $s_1, \dots, s_T$ to make the value/index relation clear.